index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:wght@400;500;600;700&display=swap');
+    </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="index.css">
+    <title>Protfolio Web</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <img class="hamburg" src="assets/hamburg.svg" alt="hamburg">
+    </nav>
+
+    <header class="header">
```

Comment:
- [ ] Kindly consider taking a `section` tag here instead of `header` for the reasons I mentioned earlier.